quantlib/termstructures/yield_term_structure.pyx

```diff
@@ -115,8 +117,10 @@ cdef class YieldTermStructure:
 
         Parameters
         ----------
-        d1, d2 : :class:`~quantlib.time.date.Date`
-            The start and end dates used to calcule the forward rate.
+        d1 : :class:`~quantlib.time.date.Date` or float
+            The start date used to calculate the forward rate.
```

Comment:
It's a time from the reference date (0. would be a frequently used for instance). d1 is not the best variable name I agree. I can be more explicit and call it date_or_time1, or specify Time in the docstring instead of float, what do you think? QuantLib Time is equivalent to float.